### ISSUE TITLE
Cant use tldr.

### ISSUE THREAD

**Issue by Darkslategray-lgtm** [denisidoro/navi#838]
**Describe the bug**
It cant use tldr and always shows 0/0 results. 
But it can use cheatsh.
Additional information: I used brew to install tldr.

**To Reproduce**

1. Input `navi --tldr docker`
2. There is 0/0 result.

**Screenshots**

![navi](https://github.com/denisidoro/navi/assets/64554933/ae906d4c-f4aa-4452-bc9c-cee620446e2a)

**Versions:**

OS: macOS
Shell: zsh
navi: 2.22.1
zsh version: zsh 5.9 (x86_64-apple-darwin22.0)

**Comment by lacygoill** [comment]
What is the output of these shell commands:

    $ type -a tldr
    $ echo $PATH
    $ tldr docker --markdown

**Comment by aserrallerios** [comment]
I have the same problem, as I described time ago in https://github.com/denisidoro/navi/issues/798.

```
❯ tldr --version
tldr v1.6.0 (v1.6.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-c-client
❯ type -a tldr
tldr is /opt/homebrew/bin/tldr
❯ tldr docker --markdown
tldr: unrecognized option `--markdown'
```

**Comment by jackandroselv** [comment]
您好！您的邮件我已收到，我会尽快的查阅邮件。谢谢您的来信！
